1.9.0

```diff
@@ -771,6 +771,186 @@
           }
         }
       ]
+    },
+    {
+      "sObjectName":"Account",
+      "description": "Add new picklist value to Source",
+      "taskList":[
+        {
+          "jiraTicket":"ac-63412a",
+          "actionType":"PicklistValueStandard",
+          "actionTypeName":"AccountSource",
+          "optionsMap":{
+            "operation" : "add",
+            "picklistValues":"Walk In,Twitter,Facebook,Social Media,Customer Referral"
+          }
+        }
+      ]
+    },
+    {
+      "sObjectName":"PersonAccount",
+      "instructionTypeName":"command__PersonAccount",
+      "taskList":[
+        {
+          "jiraTicket":"AC-63412b",
+          "actionType":"RecordTypePicklist",
+          "actionTypeName":"AccountSource",
+          "optionsMap":{
+            "operation" : "add",
+            "picklistValues":"Walk In,Twitter,Facebook,Social Media,Customer Referral"
+          }
+        }
+      ]
+    },
+    {
+      "sObjectName":"Account",
+      "instructionTypeName":"FinServ__IndustriesHousehold",
+      "taskList":[
+        {
+          "jiraTicket":"AC-63412",
+          "actionType":"RecordTypePicklist",
+          "actionTypeName":"AccountSource",
+          "optionsMap":{
+            "operation" : "add",
+            "picklistValues":"Walk In,Twitter,Facebook,Social Media,Customer Referral"
+          }
+        }
+      ]
+    },
+    {
+      "sObjectName":"Lead",
+      "description": "Add new picklist value to Source",
+      "taskList":[
+        {
+          "jiraTicket":"ac-63412c",
+          "actionType":"PicklistValueStandard",
+          "actionTypeName":"LeadSource",
```

Comment:
Change to just 'Source'.

This instruction should add these values to all the source fields on the other objects.